app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -81,7 +81,7 @@
           android:fontFamily="sans-serif-light"
           android:paddingBottom="@dimen/topic_summary_lesson_count_text_view_padding_bottom"
           android:text="@{viewModel.computeLessonCountText()}"
-          android:textColor="@color/color_def_white_80"
+          android:textColor="@color/component_color_shared_topic_card_primary_text_view_color"
```

Comment:
Remove this and use `component_color_shared_primary_text_color`

Reply:
Actually here we have to use `component_color_shared_topic_card_text_view_color` as we need white color in both dark as well as light.